src/client/client.ts

```diff
@@ -137,12 +149,33 @@ export class TravelTimeClient {
 
   routes = async (body: RoutesRequest) => this.request<RoutesResponse>('/routes', 'post', { body });
 
+  /**
+   * Simplified version of routes.
+   * Allows you to pass multiple coordinates with same params for routes to be made.
+   * @param {RoutesSimple} body Simplified RoutesSimple type that accepts multiple searches. Default search type is `departure`.
```

Comment:
I would change `multiple searches` to `multiple locations` for a more clear description

Reply:
same for all other descriptions